torch/nn/utils/rnn.py

```diff
@@ -84,7 +84,7 @@ def pack_padded_sequence(input, lengths, batch_first=False):
     return PackedSequence(torch.cat(steps), batch_sizes)
 
 
-def pad_packed_sequence(sequence, batch_first=False):
+def pad_packed_sequence(sequence, padding_value=0.0, batch_first=False):
```

Comment:
can you make it `batch_first=False, padding_value=0.0`, so that it's backward compatible.

After that, this PR is good to go. thank you.

Reply:
Thanks.
I've just committed the modified version.